torch/_tensor_str.py

```diff
@@ -254,6 +254,9 @@ def _tensor_str(self, indent):
     if self.dtype is torch.float16 or self.dtype is torch.bfloat16:
         self = self.float()
 
+    if self.dtype is torch.complex32:
```

Comment:
`self` is not actually the object for a method but argument to the function

https://github.com/pytorch/pytorch/blob/fb246140115b3d95503e46dc368de9c1674bac62/torch/_tensor_str.py#L233